### ISSUE TITLE
"Select Visible Chunks" no longer works.

### ISSUE THREAD

**Issue by NJDaeger** [llbit/chunky#475]
I'm not too sure when this happened, but when I click "Select visible chunks" it won't do anything. I don't see an error in the console either. Was hoping this could be fixed.

**Comment by llbit** [comment]
I might have time to look at this tomorrow. Thanks for the report.

**Comment by NJDaeger** [comment]
Any news?

**Comment by llbit** [comment]
I was too busy preparing for a conference to fix it, now I am traveling this week. Hopefully I'll have time next week.

**Comment by llbit** [comment]
Hi again, sorry for the delay - I have not had time to work on Chunky the past two months.

I tried to use Select Visible Chunks and it works fine for me in Chunky 1.4.5. See my [screen recording](https://imgur.com/EcUKuJ5).

Select visible chunks will select all chunks that are inside the current 2D map view bounds and inside the yellow camera view outline on the map. The yellow outline is just an estimate of what the camera could see and may not be correct: for superflat worlds with a low Y-height the camera view estimate will be too far away to "see" the closest chunks when not viewing directly down.

**Comment by chland** [comment]
I guess I know whats wrong - it seems like its working for all camera-modes except for the "Parallel"-mode. If you use that mode and click on "Select visible chunks" nothing happens.

**Comment by douira** [comment]
Can confirm this issue in Parallel camera mode.
